.github/workflows/post-release-update_bpmn_visualization_version_in_R_repo.yml

```diff
@@ -1,4 +1,4 @@
-name: Update BPMN Visualization version in R repo
+name: Update the version of the bpmn-visualization TypeScript library in R repo
```

Comment:
The content of this file matches https://github.com/process-analytics/bpmn-visualization-R/pull/172